### PR TITLE
Added the loc for task deprecated message

### PR DESCRIPTION
**Pull request by chshrikh**

### CODE REVIEW

File: Tasks/AzureWebPowerShellDeployment/task.json [hunk 1]
```diff
@@ -115,6 +115,7 @@
         "Updatingdeploymenthistoryfordeployment0": "Updating deployment history for deployment {0}",
         "Nofileswerefoundtodeploywithsearchpattern0": "No files were found to deploy with search pattern {0}",
         "Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "Found more than one file to deploy with search pattern {0}. There can be only one.",
-        "FailedtodeployWebsiteError0": "Failed to deploy Website. Error : {0}"
```
Comment (kmkumaran) [inline Tasks/AzureWebPowerShellDeployment/task.json]: Is patch version update required?

File: Tasks/AzureWebPowerShellDeployment/task.json [hunk 2]
```diff
@@ -115,6 +115,7 @@
...
+        "FailedtodeployWebsiteError0": "Failed to deploy Website. Error : {0}",
+        "DeprecatedWarning": "'Azure App Service: Classic' task will be deprecated soon. 'Azure App Service Deploy' task will replace 'Azure App Service: Classic' task and the recommendation is to migrate your Build or Release process to use the 'Azure App Service Deploy' task. Refer https://go.microsoft.com/fwlink/?LinkID=613750 for more details."
```
Comment (KrishnaAdityaB) [inline Tasks/AzureWebPowerShellDeployment/task.json]: repeating the word _task_ 4 times in a sentence is annoying. I think it is understood implicitly and you can safely delete all instances except first one.

Comment (KrishnaAdityaB) [inline Tasks/AzureWebPowerShellDeployment/task.json]: parameterize the task names with {0} and {1}. LOC team won't get confused whether to localize _classic_ or not.

Comment (chshrikh) [inline Tasks/AzureWebPowerShellDeployment/task.json]: not planning to change the warning string.

**Pull request merged**